ui/apps/platform/src/Components/ReportJob/ReportJobStatus.tsx

```diff
@@ -43,7 +43,7 @@ function ReportJobStatus({
         statusIcon = <PendingIcon title="Report run is waiting" />;
         statusText = <p>Waiting</p>;
     } else if (reportStatus.runState === 'FAILURE') {
-        statusColorClass = 'pf-v6-u-danger-color-100';
+        statusColorClass = 'pf-v6-u-text-color-status-danger';
```

Comment:
Observation: ouch, maybe we can replace class as follow with something more similar to elsewhere.

Reply:
Yikes, yes I think we need a more substantial change in this file instead of a class conversion. I'll make a ticket to do this in a follow up.